core/src/main/scala/com/rackspace/com/papi/components/checker/ByteArrayServletInputStream.scala

```diff
@@ -13,14 +13,24 @@
  *   See the License for the specific language governing permissions and
  *   limitations under the License.
  */
-package com.rackspace.com.papi.components.checker.servlet
+package com.rackspace.com.papi.components.checker
```

Comment:
Why change the package name here?  Should only be used in servlet -- not a top level class.